src/assets/style/loadingView.css

```diff
@@ -0,0 +1,68 @@
+/*#a3e3e8*/
```

Comment:
these will eventually be LESS variables, there should be a comment to that affect here